pipeline/filters/standard-output.md

```diff
@@ -1,37 +1,35 @@
-# Standard Output
+The stdout filter plugin allows printing to the standard output the data flowed through the filter plugin, which can be very useful while debugging.
 
-The **stdout** output plugin allows to print to the standard output the data received through the _input_ plugin. Their usage is very simple as follows:
+The plugin has no configuration parameters, is very simple to use.
 
-## Configuration Parameters
+## Command Line
 
-| Key | Description | default |
-| :--- | :--- | :--- |
-| Format | Specify the data format to be printed. Supported formats are _msgpack_ _json_, _json\_lines_ and _json\_stream_. | msgpack |
-| json\_date\_key | Specify the name of the date field in output | date |
-| json\_date\_format | Specify the format of the date. Supported formats are _double_,  _iso8601_ \(eg: _2018-05-30T09:39:52.000681Z_\) and _epoch_. | double |
+```
+$ bin/fluent-bit -i cpu -F stdout -m '*' -o null
```

Comment:
It is better to use `fluent-bit`.
`bin/fluent-bit` is not proper path in some environments.